go/obscuronode/nodecommon/types.go

```diff
@@ -19,6 +19,7 @@ type (
 	L2Tx                  = types.Transaction
 	EncryptedTx           []byte // A single transaction encrypted using the enclave's public key
 	EncryptedTransactions []byte // A blob of encrypted transactions, as they're stored in the rollup.
+	EncryptedParams       []byte // The encrypted params for an off-chain call or getBalance request. Encrypted with the public key of the enclave.
```

Comment:
Perhaps worth expanding here and/or on the enclave interface what this blob contains so reader doesn't have to go find the usages to imagine it (like saying it's json key-val pairs and potentially even listing what's in there since there's not many)

Reply:
:+1